src/api/message_builder/transaction.rs

```diff
@@ -42,12 +42,8 @@ pub async fn prepare_transaction(message_builder: &ClientMessageBuilder<'_>) ->
     //     .protocol
     //     .rent_structure;
     // #[cfg(not(feature = "wasm"))]
```

Comment:
The commented part here should be moved into get_byte_cost_config (uncommented, no need to wait for difficulty)